add PoE documentation (#71)
bisdn/bisdn.github.io: platform_configuration/power_over_ethernet.md

Two types of devices supporting PoE are considered regarding PoE capabilities: the *Power Sourcing Equipment (PSE)*, which are the devices capable of providing power on the cable; and the *Powered Device (PD)*, which are devices that consume energy provided by PoE. Some BISDN Linux supported switches can act as a Power Sourcing Equipment (please refer to the platform specification provided by your vendor).  

## Configuration

BISDN Linux comes with the command line utility `poectl`, which provides a simple and convenient way to configure PoE.

### Usage

```
:~$ poectl -h
poectl for enable, disable, status and measurements of PoE ports
 
 Usage:  poectl [OPTIONS] PORT | all
 
 OPTIONS:
  -h   help
  -v   version
  -e   enable PoE
  -d   disable PoE
  -s   PoE status
  -mV  PoE voltage
  -mC  PoE current
  -mT  PoE temperature
  -mP  PoE power
```

### Examples

> To enable PoE on a port run the following command:

Reviewer: ```suggestion
To enable PoE on all ports run the following command:
```

```

> :~$ poectl -e port2

Reviewer: ```suggestion
:~$ poectl -e
```

Author: While this does work, the help usage says you should use `all`, so we should stick with that.

Reviewer: ```suggestion
:~$ poectl -e all
```

```

> To disable PoE on a port:

Reviewer: ```suggestion
To disable PoE on a certain port:
```
